Dockerfile

```diff
@@ -1,10 +1,9 @@
-# impish is needed since glibc 2.34 is required for linux 5.15.30
-FROM ubuntu:impish-20220316
+FROM ubuntu:jammy-20220815
 
 # according to https://www.raspberrypi.com/documentation/computers/linux_kernel.html#cross-compiling-the-kernel
 RUN apt-get update && \
     apt-get install -y  git bc bison flex libssl-dev make libc6-dev libncurses5-dev \
-                        crossbuild-essential-armhf
+                        crossbuild-essential-armhf kmod
 
 # to compile go programs
 RUN apt-get -y install golang-go gcc-arm-linux-gnueabi
```

Comment:
Since you are adjusting the Dockerfile, could you remove those lines as well?

```suggestion
```

I added them because I wanted to cross-compile grafana, but [I ended up](https://github.com/oliverpool/grafana-armv6/) using https://github.com/Pro/raspi-toolchain, because [newer GCC versions do not create correct binaries for ARMv6](https://github.com/gokrazy/tools/issues/32#issuecomment-1082779767)

Reply:
(I should probably add a comment in the readme about this)